server/bundles/io.cloudbeaver.service.security/src/io/cloudbeaver/service/security/CBEmbeddedSecurityController.java

```diff
@@ -2202,7 +2232,8 @@ private String findOrCreateExternalUserByCredentials(
             }
             Object reverseProxyUserTeams = sessionParameters.get(SMConstants.SESSION_PARAM_TRUSTED_USER_TEAMS);
             if (reverseProxyUserTeams instanceof List) {
-                setUserTeams(userId, ((List<?>) reverseProxyUserTeams).stream().map(Object::toString).toArray(String[]::new), userId);
+                List<String> reverseProxyUserTeamsList = (List<String>) reverseProxyUserTeams;
```

Comment:
redundant change?

Reply:
no, here we added default team for proxyUsers, before that users didnt have deafult group